bobber/lib/tests/run_tests.py

```diff
@@ -83,13 +83,17 @@ def run_stg_bw(args: Namespace, bobber_version: str, iteration: int,
                               f'threads_{args.bw_threads}_'
                               f'direct_{args.direct}_'
                               f'depth_{args.io_depth}_'
+                              f'read_pattern_{args.read_pattern}_'
+                              f'write_pattern_{args.read_pattern}_'
```

Comment:
This doesn't change anything with our parser, correct? I don't think it should as it shouldn't expect a rigid structure for the log's filename, but something I'd want to double check.

Reply:
Still need to test - worst case, will try to test locally on my Mac.

Reply:
Let me know if you want me to run on my workstation and I can test it out too.